tools/sklearn/search_model_validation.xml

```diff
@@ -551,7 +667,118 @@ The evaluation of settings supports operations in Math, list comprehension, nump
 
 - max_depth: scipy_stats_randin(1, 11)
 
-- estimator: [ensemble_ExtraTreesClassifier(n_estimators=100, random_state=324089)]
+**Estimator search/eval (additional '-'):**
+
+- base_estimator-: [sklearn_tree.DecisionTreeRegressor(), sklearn_tree.ExtraTreeRegressor()]
```

Comment:
I find this `-:` syntax very confusing. Can you explain this in more detail, please.

Reply:
No better reason, just differ from other non-estimator eval. Do you have better idea?